QA/flag_test.md

```
@@ -0,0 +1,107 @@
+go run debug_cli.go export --select
+```
+
+この診断ツールは、フラグの解析過程と結果を詳細に表示します。
+
+## 問題修正のヒント
+
+もし環境特有の問題でフラグが認識されない場合は、以下の方法で問題を回避できる可能性があります：
+
+1. 短いフラグ形式を試す: `--select` の代わりに `-select` を使用
+2. 値付きフラグを試す: `--select=true` の形式を使用
+3. フラグの順序を変更する: 例えば `--select --new-shell` の代わりに `--new-shell --select` を試す
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)